packages/files-ui/src/Components/Modules/FileBrowsers/views/FileSystemItem/FileSystemGridItem.tsx

```diff
@@ -196,8 +199,11 @@ const FileSystemGridItem = React.forwardRef(
       onSubmit: (values: { name: string }) => {
         const newName = extension !== "" ? `${values.name.trim()}.${extension}` : values.name.trim()
 
-        if (newName !== name) {
-          newName && handleRename && handleRename(file.cid, newName)
+        if (newName !== name && newName && handleRename) {
```

Comment:
To align with the rest
```suggestion
        if (newName !== name &&!! newName && handleRename) {
```